### ISSUE TITLE
Docs for 'port' attribute in junos_* modules are wrong/incomplete

### ISSUE THREAD

**Issue by agaffney** [ansible/ansible#17076]
##### ISSUE TYPE

- Documentation Report
##### COMPONENT NAME

junos_command (and related modules)
##### ANSIBLE VERSION

```
ansible 2.1.0.0
```
##### SUMMARY

The common `junos` module documentation says the following for the `port` parameter:

```
Specifies the port to use when buiding the connection to the remote device. The port value will default to the well known SSH port of 22
```

However, this is true only for `transport=cli`. When using `transport=netconf` (which seems to be the only option for many of the `junos_*` modules), the default port is 830 (from the code).

**Comment by agaffney** [comment]
PR to fix:

https://github.com/ansible/ansible/pull/17077

**Comment by bcoca** [comment]
fixed by merge of #17077
